client/src/infographic/Infographic.js

```diff
@@ -422,8 +422,18 @@ const Infographic = ({
       <Redirect to={infograph_href_template(subject_parent, bubble_id, "/")} />
     );
   }
-
   const title = text_maker("infographic_for", { subject });
+  const breadcrumb_program =
+    level === "program"
+      ? [
+          {
+            title: text_maker("department_title", { subject }),
+            department: true,
+            id: subject.dept.id,
+          },
+          { title: text_maker("infographic_title", { subject }) },
+        ]
+      : [{ title: text_maker("infographic_title", { subject }) }];
```

Comment:
There could potentially be a better way to format this.

Reply:
Was there a reason for putting them thru yaml? You can just do `subject.name` directly

Reply:
Since it was originally using a text_maker function, I decided to follow the format. I'll do it like how you suggested!